src/Data/CallbackRequest/Pisa/pisaCallbackRequestDataClass.ts

```diff
@@ -0,0 +1,22 @@
+import { provideDataClass } from 'scrivito'
+import { pisaClient } from '../../pisaClient'
+import { DataIndexResponse, RawItem } from '../../types'
+
+export async function pisaCallbackRequestDataClass() {
+  const callbackRequestClient = await pisaClient('callback-request')
+
+  // callback-request is more or less a "singleton". It only offers PUT, GET and DELETE.
```

Comment:
* [ ] FYI This is a weird API. It would be great if we made this more standard because it would reduce the complexity on the consuming side.

Concurrency aside, our list of callback requests would be zero or one in length anyway. The `localStorageCallbackRequestDataClass` implementation proves this.